fridgify/lib/data/content_repository.dart

```diff
@@ -55,15 +58,20 @@ class ContentRepository implements Repository<Content> {
       var c = jsonDecode(response.body);
 
       logger.i("ContentRepository => CREATED SUCCESSFUL $c");
-
-      return fridge.fridgeId;
+      for(var con in c) {
```

Comment:
I had some tests failing because my mocked object couldnt be iterated over. I needed to change the object to an array of objects but after checking with the swagger documentation i didnt find anything that would suggest an returned array. Im talking about the POST request on fridge/content. Is the implementation or the documentation false ? @DonkeyCo / @DennisRein

Reply:
After thinking about it, I see a mistake in the backends implementation. Currently only the created item is being returned, but you are absolutely correct in stating that a list of content should be returned. If you want to, you can create an issue and assign me and I will fix it tomorrow.

Reply:
Alright will do.

Reply:
I was actually receiving a list of contents, that was the reason I redid this part. It came with the count changes. Which allows the addition of multiple items at once.

Reply:
Well maybe the swagger spec is missing something then ?

Reply:
Yep. You're right. Just had a look at the code and its just a faulty Swagger Spec